### ISSUE TITLE
Element on the screen should be found

### ISSUE THREAD

**Issue by sguptatw** [getgauge/taiko#186]
**Expected behavior**
Should be able to work on an element visible on the UI

**Actual behavior**
Getting `Combo Box not found`

**Steps to replicate**
* Run the script
```
const { openBrowser, goto, toRightOf, checkBox } = require('taiko');

(async () => {
    try {
        await openBrowser({headless:false});
        await goto("https://react-shopping-cart-67954.firebaseapp.com/");
        await comboBox(toRightOf(text(contains("Order by")))).select("Lowset to highest");
    } catch (e) {
        console.error(e);
    } finally {
        closeBrowser();
    }
})();
```

**Version**
```
commit - 09afc93da99da144aa002a91feb1bd4b44506010
```

**Comment by NivedhaSenthil** [comment]
The combo box is within the text element.
```
<div class="sort">
Order by 
<select><option value="">Select</option><option value="lowestprice">Lowest to highest</option><option value="highestprice">Highest to lowest</option></select>
</div>
```

 As both are same element and there is no other element matching `toRightOf` taiko gives `combo box not found`. Changing the script to use `near` works,
`await comboBox(near(text(contains("Order by")))).select("Lowest to highest");`

**Comment by sguptatw** [comment]
IMO - Since it is a UI element and not text itself, the script `await comboBox(near(text("Order by"))).select("Lowest to highest");` should work and should not expect `contains`. 

Thoughts?

**Comment by NivedhaSenthil** [comment]
Currently this is happening because the default behaviour is to search for text across elements.

**Comment by Apoorva-GA** [comment]
Since `Order by` is an atomic text node here. There is no way we can get the position of the same.
One option was to edit the DOM to wrap the atomic text node in a `div` tag and get the position and delete the same. 
But this will cause problems with respect to the positions.

**Comment by NivedhaSenthil** [comment]
Fix available in 7d43b16

**Comment by Debashis9012** [comment]
This issue has been verified and found fixed with the version ```7d43b16 ```
**Note:** Contains has been deprecated and text api will be doing the partial text match.
Kindly find the below correct piece of code .

``` comboBox(toRightOf(text("Order by"))).select("Lowest to highest"); ```
